assignment-section-05/generate_test_data.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+
+from classification_model.config.core import config, DATASET_DIR
+from classification_model.processing.data_manager import _load_raw_dataset
+
+
+def generate_data_data() -> None:
+    """
+    Generate train.csv and test.csv files.
+
+    The titanic dataset does not come split into train and test files, so we
+    do this ourselves. Note, this requires you to have downloaded the original
+    datafile from here: https://www.openml.org/data/get_csv/16826755/phpMYEkMl
+    and renamed it to match your config raw_data_file name.
```

Comment:
the instruction to download the dataset is too late here. By now, they would have completely refactored the code. This needs to be done before they start the assignment.